### ISSUE TITLE
Drawer - controls should be at the top and focused first

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt-contrib-core#101]

**Comment by joe-replin** [comment]
## Bug Description:
The Back and Close buttons of the resources drawer are placed illogically in the keyboard focus order. Currently, these controls are placed at the top of the drawer, but are placed at the end of the focus order.

## Steps to Reproduce:

    Using a keyboard, activate the resources button.
    Activate the Resources option, then tab through the contents of the drawer, noting the order of focus. 

## Actual Behavior:

See bug description

## Expected Behavior:

Ensure keyboard focus follows a logical left to right, top to bottom focus order.
